src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,55 @@
+.nav {
+  &__list {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+  }
+
+  &__item {
+    margin-right: 20px;
+    display: flex;
+  }
+
+  &__link {
+    display: inline-flex;
+    align-items: center;
+    min-height: 60px;
+
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $color-dark;
+    font-size: 12px;
+    cursor: pointer;
+    transition: color 0.3s;
+    font-weight: 500;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link:hover {
+    color: $color-blue;
+  }
+
+  .is-active {
+    position: relative;
+    color: $color-blue;
+  }
+
+  .is-active::after {
+    position: absolute;
+    content: "";
+    display: block;
+
+    width: 100%;
+    height: 4px;
+
+    background-color: $color-blue;
+    border-radius: 8px;
+
+    left: 0;
+    top: 56px;
+  }
```

Comment:
Here also you should put pseudo-class ::after inside .is-active and name it &::after. With &__item and &__link the same.